nemo/collections/nlp/models/text_classification_model.py

```diff
@@ -60,28 +91,35 @@ def __init__(
             class_balancing: enables the weighted class balancing of the loss, may be used for handling unbalanced classes
         """
 
+        self.max_seq_length = cfg.language_model.max_seq_length
+        self.data_dir = cfg.data_dir
+        self.tokenizer = NemoBertTokenizer(pretrained_model=cfg.language_model.pretrained_model_name)
```

Comment:
@VahidooX in a future PR we should change to use tokenizer utils to support sentence piece tokenizer as well